conf/logback.xml

```diff
@@ -14,19 +14,6 @@
         </encoder>
     </appender>
 
-    <appender name="STDOUT_IGNORE_NETTY" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%date{ISO8601} level=[%level] logger=[%logger] thread=[%thread] rid=[not-available] user=[not-available] message=[%message] %replace(exception=[%xException]){'^exception=\[\]$',''}%n</pattern>
```

Comment:
we had a story in the [past](https://jira.tools.tax.service.gov.uk/browse/MISUV-7355) to add source of the error to the logs, this might be specific to our project, but removing these (even to satisfy PR Bot warning) would cause problem with investigation

Reply:
This comment has been discussed with the resolution at this comment on the frontend: https://github.com/hmrc/income-tax-view-change-frontend/pull/2493#discussion_r1742352671